keyboards/waterfowl/info.json

```
@@ -0,0 +1,54 @@
+{
+    "keyboard_name": "Waterfowl",
+    "url": "",
+    "maintainer": "CyanDuck",
```

This field is meant to reference your GitHub username.

```suggestion
    "maintainer": "JW2586",
```